models/users.js

```diff
@@ -178,26 +178,39 @@ const fetchPaginatedUsers = async (query) => {
   }
 };
 
-const fetchAllUsers = async () => {
+const fetchFilteredUsers = async (usernames = []) => {
   try {
     const dbQuery = userModel;
+    const filterdUsersWithDetails = [];
 
-    const snapshot = await dbQuery.get();
+    const groups = [];
+    const batchSize = 30; // since only 30 comparisons are allowed with the 'in' clause
```

Comment:
@khalatevarun Would be nice to move this to a "Firestore Constants" file, so we have a single source of truth for all knowledge and usage to things related to Firestore specifics. That way, in future, let's say batch size changes to 50 in a new firestore version, then we can just update the config constant in one place, and it will apply throughout the codebase.

Reply:
sure, makes sense
have moved the constant to firebase.js under constants directory